File: beginner/__2000__/2759/2759.cpp
#include <iostream>
#include <vector>
#include <string>

using namespace std;

int main(){
    int index = 0;
    string curr;

    vector<string> data;

    getline(cin, curr);
    data.push_back(curr);

    getline(cin, curr);
    data.push_back(curr);

    getline(cin, curr);
    data.push_back(curr);

    for(int i = 0; i < 3; i++){
        string A = data[index];
        string B = data[(index + 1) % data.size()];
        string C = data[(index + 2) % data.size()];

        cout << "A = " << A << ", B = " << B << ", C = " << C << endl;
        index = (index + 1) % data.size();
    }

    return 0;
}

File: beginner/__2000__/2759/2759.py
a = input()
b = input()
c = input()

index = 0
data = [a, b, c]

for _ in range(3):
    A = data[index]
    B = data[(index + 1) % len(data)]
    C = data[(index + 2) % len(data)]
    print(f"A = {A}, B = {B}, C = {C}")
    index = (index + 1) % lne(data)